snap/snapcraft.yaml

```diff
@@ -50,6 +51,19 @@ parts:
     organize:
         "usr/bin/fuse-overlayfs": "libexec/rockcraft/fuse-overlayfs"
 
+  libgit2: # part came from snapcraft@5fbaab144842f
+    source: https://github.com/libgit2/libgit2/archive/refs/tags/v1.7.1.tar.gz
```

Comment:
Shallow-cloning the repository at this specific tag could be better than downloading the tarball.